config.example.yaml

```diff
@@ -1,11 +1,35 @@
 address: 127.0.0.1:8888
 
-log:
-  format: json
-  level: info
-
 telemetry:
-  address: 127.0.0.1:10000
+  log:
+    level: debug
+
+sink:
+  kafkaConnect:
+    enabled: true
+    url: http://kafka-connect:8083
+    connectors:
+      - name: clickhouse
+        type: clickhouse
+        config:
+          hostname: clickhouse
+          port: 8123
+          ssl: false
+          database: openmeter
+          username: default
+          password: default
+          deadLetterQueue:
+            topicName: om_deadletterqueue
+            replicationFactor: 1
+            contextHeaders: true
+
+# dedupe:
```

Comment:
why commented out?

Reply:
use the memory dedup as default?

Reply:
yep

It's there so we can easily enable it locally. But locally, memory dedupe is the default.